keyboards/keyten/kt60_mu/kt60_mu.c

```diff
@@ -0,0 +1,12 @@
+// Copyright 2024 QMK Collaborators (@qmk)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "quantum.h"
+
+#define LED_INDICATOR_PIN B11
+
+void matrix_init_kb(void) {
+    gpio_set_pin_output(LED_INDICATOR_PIN);
+    gpio_write_pin_high(LED_INDICATOR_PIN);
+    matrix_init_user();
+}

```

Comment:
Should fix lint error.
```suggestion
// Copyright 2024 QMK Collaborators (@qmk)
// SPDX-License-Identifier: GPL-2.0-or-later

#include "quantum.h"

#define LED_INDICATOR_PIN B11

void matrix_init_kb(void) {
    gpio_set_pin_output(LED_INDICATOR_PIN);
    gpio_write_pin_high(LED_INDICATOR_PIN);
    matrix_init_user();
}
```

Reply:
Hmm
![image](https://github.com/user-attachments/assets/094bdd5a-2507-4fd4-96d6-7f8e29938cc0)

Reply:
```
❯ file keyboards/keyten/kt60_mu/kt60_mu.c
keyboards/keyten/kt60_mu/kt60_mu.c: C source, ASCII text, with CRLF line terminators
```
You need to use LF line terminators in the repo.

Reply:
I can't do this using GitHub web editor...